languages/kotlin/app/build.gradle

```diff
@@ -48,7 +48,7 @@ android {
 }
 
 dependencies {
-    // implementation 'com.bitwarden.sdk'
+    // implementation 'com.bitwarden.sdk-android'
```

Comment:
⚠️ This is in the `kotlin` directory which is why I named my PR that way, but if `android` makes sense then fine by me.

Reply:
I think it does, the artifact is android specific. We should probably consider renaming the directory.